<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 179, 90);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Reservation</string>
      </property>
      <widget class="QLabel" name="nom1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="prenom1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="age1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>num_reserv</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="num">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="cin1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date de debut</string>
       </property>
      </widget>
      <widget class="QLabel" name="referance1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cin">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="email1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>EMAIL</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>260</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="date">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date fin</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="datef">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>durer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>330</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>supprimer</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>supprimer Reservation</string>
      </property>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>num_reserv</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="num_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>modifier</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier Reservation</string>
      </property>
      <widget class="QLabel" name="nom1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>num_reserv</string>
       </property>
      </widget>
      <widget class="QLabel" name="prenom1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="age1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="num_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="cin1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date de debut</string>
       </property>
      </widget>
      <widget class="QLabel" name="referance1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cin_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="email1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>EMAIL</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>260</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="date_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>51</y>
        <width>361</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherchecin">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher cin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Trie_name">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>tri nom</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>221</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Imprimer PDF</string>
      </property>
      <widget class="QLineEdit" name="PDF">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_pdf">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>260</y>
        <width>221</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Exporter Excel</string>
      </property>
      <widget class="QPushButton" name="export_button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>EXCEL</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLineEdit" name="from">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>395</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>395</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>391</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendMail">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mailsent">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_78">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>De</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_41">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_108">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Objet</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_110">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Corps du mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="to">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>395</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="paswd">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>395</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1199</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
